src/components/NestedEdgeList.svelte

```diff
@@ -30,8 +31,8 @@
 		open_signal = null;
 	}
 </script>
-
-{#each tree.sort((a, b) => sort(a.edge, b.edge)) as item, i}
+<!-- TODO add sorting using the edge sorter -->
```

Comment:
Why not do the sorting on JS side? Will doing it in Rust be significantly faster? Probably, huh